pkg/provision/interface.go

```diff
@@ -123,7 +123,7 @@ type Storage interface {
 	// ByTwin return list of deployments for a twin
 	ByTwin(twin uint32) ([]uint64, error)
 	// return total capacity and active deployments
-	Capacity() (cap gridtypes.Capacity, active []gridtypes.Deployment, err error)
+	Capacity() (cap gridtypes.Capacity, active []gridtypes.Deployment, workloads int, err error)
```

Comment:
This returned argument set is becoming too long. May be create a `StorageCapacity` struct (in this file) and then change the interface to 
```go
Capacity() (StorageCapacity, error)
```